proto/accessgraph/v1alpha/query.proto

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package accessgraph.v1alpha;
+
+import "teleport/legacy/client/proto/event.proto";
+import "teleport/legacy/types/types.proto";
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/accessgraph/v1alpha;accessgraphv1alpha";
+
+// AccessGraphService is a service for interacting the access graph service.
+service AccessGraphService {
+  // Query queries the access graph.
+  // Currently only used by WebUI.
```

Comment:
I'm not sure this kind of comment belongs here, as ideally the RPCs are meant to be widely useful and fulfill a complete set of system operations.